lib/senders/sms.js

```diff
@@ -162,3 +162,20 @@ function urlSafeBase64 (hex) {
     .replace(/\//g, '_')
     .replace(/=/g, '')
 }
+
+function startOfMinute (date) {
```

Comment:
Is it worth a few quick testcases for this, or are they already covered by existing tests?

Reply:
I'm pretty comfortable with the coverage as it is but more tests can't hurt, so I'll add some.